.scalafmt.conf

```diff
@@ -16,6 +17,10 @@ align {
   openParenDefnSite = false
 }
 
+runner {
+  dialect = scala212
```

Comment:
Curious, I know this is just a formatting thing but seeing 212 instead of 213 gave me a double take. 😅 Just an FYI for the future, personally prefer if formatting commits were separate and done after any actual code changes, it makes a PR easier to follow. 👍